models/cxr_image_synthesis_latent_diffusion_model/large_files.yml

```
@@ -0,0 +1,9 @@
+large_files:
+  - path: "models/autoencoder.pt"
+    url: "https://drive.google.com/uc?export=download&id=1paDN1m-Q_Oy8d_BanPkRTi3RlNB_Sv_h"
+    hash_val: "7f579cb789597db7bb5de1488f54bc6c"
+    hash_type: "md5"
+  - path: "models/model.pt"
+    url: "https://drive.google.com/uc?export=download&id=1CjcmiPu5_QWr-f7wDJsXrCCcVeczneGT"
```

```suggestion
    url: "https://drive.google.com/uc?id=1paDN1m-Q_Oy8d_BanPkRTi3RlNB_Sv_h"
    hash_val: "7f579cb789597db7bb5de1488f54bc6c"
    hash_type: "md5"
  - path: "models/model.pt"
    url: "https://drive.google.com/uc?id=1CjcmiPu5_QWr-f7wDJsXrCCcVeczneGT"
```

I think one issue is that the URLs are malformed. Gdown will pull down the html page saying something was wrong rather than error out. You can try the URLs on the command line with `gdown` then check their hashes with `md5sum`.